driver-core/src/test/java/com/datastax/driver/core/policies/AbstractRetryPolicyIntegrationTest.java

```diff
@@ -113,18 +114,18 @@ protected void simulateError(int hostNumber, Result result) {
     protected void simulateError(int hostNumber, Result result, Config config) {
         PrimingRequestBuilder builder = PrimingRequest.queryBuilder()
                 .withQuery("mock query")
-                .withResult(result);
+                .withThen(then().withResult(result));
 
         if (config != null)
-            builder = builder.withConfig(config);
+            builder = builder.withThen(then().withConfig(config));
```

Comment:
All the failures were children of `AbstractRetryPolicyIntegrationTest` ran through them and it does appear that this fixes them all.